Analysis/include/SampleDescriptor.h

```diff
@@ -39,14 +41,35 @@ struct AnalyzerSetup {
     std::string plot_cfg, plot_page_opt, unc_cfg;
     JetOrdering jet_ordering;
     std::map<Channel, std::vector<std::string>> trigger;
-
     std::map<SelectionCut,analysis::EllipseParameters> massWindowParams;
+    std::map<std::string, std::vector<std::string>> limit_setup_raw;
 
     bool IsSignal(const std::string& sample_name) const
     {
         const auto iter = std::find(signals.begin(), signals.end(), sample_name);
         return iter != signals.end();
     }
+
+    void CreateLimitSetups()
+    {
+        std::map<EventCategory, std::string> second_map;
+        for(const auto& item: limit_setup_raw) {
+            const auto& setup = item.first;
+            for (size_t i = 0; i < 2; i++){
+                auto second_list = SplitValueList(item.second.at(i), false, ":");
```

Comment:
use more specific name rather than "second_list"

Reply:
I was struggling with this. What do you think of variable_category?

Reply:
yes, better. I suggest to use a plural, i.e. `variable_categories`.